sdk/servicebus/Azure.Messaging.ServiceBus/samples/TopicFilters/README.md

```diff
@@ -0,0 +1,69 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-service-bus
+name: Explore topic filters in Azure Service Bus
+description: This sample shows how to apply topic filters to subscriptions.
+---
+
+# 
+
+This sample shows how to apply topic filters to subscriptions. 
+
+## What is a Topic filter?
+
+[Topic filters](https://docs.microsoft.com/azure/service-bus-messaging/topic-filters), or rules, can be applied to subscriptions to allow subscribers to define which messages they want to receive from a topic.
+
+## Sample Code
+
+The sample implements four scenarios:
+
+* Create a subscription with no filters. Technically, all subscriptions are created with the default `TrueFilter`. In the sample, we remove and re-add this filter.
```

Comment:
This covers some of my thoughts from the sample code itself.   It may be helpful to bridge the gap and either reflect a small part of this or make a reference back to "see the README description for sample 3" in the code itself to aid understanding.